modules/builds-strategy-docker-build-arguments.adoc

```diff
@@ -24,7 +24,7 @@ dockerStrategy:
     - name: "foo"
```

Comment:
foo should be avoided: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#c:~:text=the%20following%20functions%E2%80%9D.-,foo,-noun ISG